activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Fixed odd behavior of inverse_of with multiple belongs_to to same class #35204
```

Comment:
```suggestion
*   Fix odd behavior of inverse_of with multiple belongs_to to same class.

    Fixes #35204.
```

Ref https://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#updating-the-changelog

Reply:
Thank you. I fixed.